.mocharc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   recursive: true,
   reporter: 'dot',
-  require: [require.resolve('@babel/register'), require.resolve('./test/utils/setup')],
+  require: [require.resolve('@babel/register'), require.resolve('./test/utils/setupJSDOM')],
```

Comment:
It's really about setting up JSDOM. I was always confused by the two `init` and `setup` files which are essentially synonyms in this context.

Reply:
@dependabot squash and merge